packages/aws-cdk-lib/aws-dynamodb/README.md

```diff
@@ -1,240 +1,748 @@
 # Amazon DynamoDB Construct Library
 
+> The DynamoDB construct library has two table constructs - `Table` and `GlobalTable`. `GlobalTable` is the preferred construct to use for creating a single table or a table with multiple `replicas`. A `GlobalTable` without any `replicas` configured will create a single table in the primary deployment region and will behave in the same way as the `Table` construct.
 
-Here is a minimal deployable DynamoDB table definition:
+[`Table` API documentation](./ORIGINAL_API.md)
+
+Here is a minimal deployable DynamoDB `GlobalTable` definition:
 
 ```ts
-const table = new dynamodb.Table(this, 'Table', {
-  partitionKey: { name: 'id', type: dynamodb.AttributeType.STRING },
+const globalTable = new dynamodb.GlobalTable(this, 'GlobalTable', {
+  partitionKey: { name: 'pk', type: dynamodb.AttributeType.STRING },
+});
+```
+
+By default, a `GlobalTable` will have one primary replica table in the main deployment region. The properties of the replica table in the main deployment region are configurable via the `GlobalTable` properties. For example, consider the following `GlobalTable` defined in a `Stack` being deployed to `us-west-2`:
+
+```ts
+const globalTable = new dynamodb.GlobalTable(this, 'GlobalTable', {
+  partitionKey: { name: 'pk', type: dynamodb.AttributeType.STRING },
+  contributorInsights: true,
+  tableClass: dynamodb.TableClass.STANDARD_INFREQUENT_ACCESS,
+  pointInTimeRecovery: true,
+});
+```
+
+The above `GlobalTable` definition will result in the provisioning of a single table in `us-west-2` with properties that match what was defined on the `GlobalTable`.
+
+Further reading:
+https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GlobalTables.html
+
+## Replicas
+
+A `GlobalTable` can be configured with replica tables. To do this, the `GlobalTable` must be defined in a region non-agnostic `Stack`. The main deployment region must not be given as a replica because this is created by default with the `GlobalTable`. The following is a minimal `GlobalTable` definition with `replicas` defined in `us-east-1` and `us-east-2`:
```

Comment:
Minor minor: negatives are annoying to parse. Try to avoid them.

> region non-agnostic Stack

⇒

> Stack with a defined region

Reply:
Also, make sure in the prose there will be 3 copies of the table, one in the primary region and 2 replicas.

Reply:
Thanks. Updated!